src/deezer/resources/resource.py

```diff
@@ -64,75 +64,51 @@ def get_relation(self, relation, **kwargs):
         is not meant to be used directly by a client, it's more
         a helper method for the child objects.
         """
-        return self.client.request(
+        return await self.client.request(
             "GET",
             f"{self.type}/{self.id}/{relation}",
             parent=self,
             **kwargs,
         )
 
-    def post_relation(self, relation, **kwargs):
+    async def post_relation(self, relation, **kwargs):
         """
         Generic method to make a POST request to a relation from any resource.
 
         This is not meant to be used directly by a client, it's more
         a helper method for the child objects.
         """
-        return self.client.request(
+        return await self.client.request(
             "POST",
             f"{self.type}/{self.id}/{relation}",
             **kwargs,
         )
 
-    def delete_relation(self, relation, **kwargs):
+    async def delete_relation(self, relation, **kwargs):
         """
         Generic method to make a DELETE request to a relation from any resource.
 
         This is not meant to be used directly by a client, it's more
         a helper method for the child objects.
         """
-        return self.client.request(
+        return await self.client.request(
             "DELETE",
             f"{self.type}/{self.id}/{relation}",
             **kwargs,
         )
 
-    def get_paginated_list(
+    async def get_paginated_list(
         self,
         relation: str,
         **kwargs,
     ):
-        return PaginatedList(
+        paginated_list: PaginatedList = PaginatedList(
             client=self.client,
             base_path=f"{self.type}/{self.id}/{relation}",
             parent=self,
             **kwargs,
         )
-
-    def __getattr__(self, item: str) -> Any:
-        """
-        Called when the default attribute access fails with an AttributeError.
-
-        This is a fallback method, not need to call the parent implementation.
-        If the attribute is found through the normal mechanism, this is NOT called.
-        """
-        class_annotations = self.__class__.__annotations__
-        if item in class_annotations:
-            result = self._infer_missing_field(item)
-            if result is not NOT_INFERRED:
-                setattr(self, item, result)
-                self._fields += (item,)
-                return result
-            elif not getattr(self, "_fetched", False):
-                full_resource = self.get()
-                missing_fields = set(full_resource._fields) - set(self._fields)
-                for field_name in missing_fields:
-                    setattr(self, field_name, getattr(full_resource, field_name))
-                    self._fields += (field_name,)
-                return getattr(self, item)
-        raise AttributeError(
-            f"'{self.__class__.__name__}' object has no attribute '{item}'"
-        )
+        return await paginated_list.fetch()
```

Comment:
So we fetch by default 500 items, right?

Reply:
Yes, but now when i think about it we shouldnt limit it at all. Since deezers limit on tracks in playlists is 2000... Also in the fetch function i should also use hasattr instead of try catch :wink:

Reply:
I have now increased it to 99999

Reply:
Wait oh god the limit of favorite tracks is 10 000.... I guess i will increase it again.................